src/PrestaShopBundle/Form/Admin/Sell/Order/Invoices/InvoiceOptionsType.php

```diff
@@ -78,36 +80,164 @@ public function __construct(
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $featureTranslationDomain = 'Admin.Orderscustomers.Feature';
+        $helpTranslationDomain = 'Admin.Orderscustomers.Help';
+
         $builder
-            ->add('enable_invoices', SwitchType::class)
-            ->add('enable_tax_breakdown', SwitchType::class)
-            ->add('enable_product_images', SwitchType::class)
-            ->add('invoice_prefix', TranslatableType::class, [
-                'type' => TextType::class,
-            ])
-            ->add('add_current_year', SwitchType::class)
-            ->add('reset_number_annually', SwitchType::class)
-            ->add('year_position', ChoiceType::class, [
-                'choices' => [
-                    $this->trans('After the sequential number', 'Admin.Orderscustomers.Feature') => 0,
-                    $this->trans('Before the sequential number', 'Admin.Orderscustomers.Feature') => 1,
-                ],
-                'expanded' => true,
-            ])
-            ->add('invoice_number', NumberType::class, [
-                'required' => false,
-            ])
-            ->add('legal_free_text', TranslatableType::class, [
-                'type' => TextareaType::class,
-            ])
-            ->add('footer_text', TranslatableType::class, [
-                'type' => TextType::class,
-            ])
-            ->add('invoice_model', ChoiceType::class, [
-                'choices' => $this->invoiceModelChoiceProvider->getChoices(),
-                'translation_domain' => false,
-            ])
-            ->add('use_disk_cache', SwitchType::class);
+            ->add(
+                'enable_invoices',
+                SwitchType::class,
+                [
+                    'required' => true,
+                    'multistore_configuration_key' => 'PS_INVOICE',
+                    'label' => $this->trans('Enable invoices', $featureTranslationDomain),
+                    'help' => $this->trans(
+                        'If enabled, your customers will receive an invoice for the purchase.',
+                        $helpTranslationDomain
```

Comment:
I know it's counter-intuitive but you shouldn't use a variable to indicate the domain for two reasons:
- we have a tools that allow us to scan the code in PrestaShop it searches the usage of the `trans` function and is able to extract the wording and namespace, which are then used to generate our XLF catalog files But the tool is not able to interpret variables so it won't know which domain this wording belongs to
- we also have a bot that is able to detect new wordings in the PR content, similarly it is also not able to detect the namespace if a variable is used, this one seems to work better because it detected that you are using `Admin.Orderscustomers.Feature` but either it's buggy or it found a namespace somewhere else but anyway the proper namespace should be `'Admin.Orderscustomers.Help'` so something seems off https://github.com/PrestaShop/PrestaShop/pull/28825#issuecomment-1164182067

Anyway although the idea was good because of those limitations we always hard-code the wording and namespace parameters when we use `trans`

Reply:
For the second point, it probably detected the `'translation_domain' => 'Admin.Orderscustomers.Feature'` on L257
But you can't replace this one either because you are using different wordings which are in different namespaces, so a single configuration is not enough anyway I'm not event sure this option is really usable here if we explicitly provide the namespace for all the wordings used here

Reply:
Be careful to use the appropriate namespace, all these wordings are existing so if you search for them in the project you will easily find in which XLF catalog they are and which namespace to use

Reply:
Thanks for the info, I would never find this by my own xD ^^'